build/yamls/windows/containerd/agent-containerd.yml

```diff
@@ -4,7 +4,6 @@ metadata:
   labels:
     component: antrea-agent
   name: antrea-agent-windows
-  namespace: kube-system
```

Comment:
Why do we need this?

Reply:
I think this is a redundant value. I got error `no matches for Id DaemonSet.v1.apps/antrea-agent-windows.[noNs]; failed to find unique target for patch DaemonSet.v1.apps/antrea-agent-windows.[noNs]`without change.